src/routes/blog/post/How-to-put-privacy-first/+page.markdoc

```diff
@@ -0,0 +1,86 @@
+
+If you want to use data for decision-making or personalizing the user experience, removing tags, pixels, and cookies might not be your first choice. But as mentioned, we are here to challenge this idea. Do you really need to track all this personal data to do your work in growth? We think not. Since there are many privacy-friendly options for getting the data and insights you need, and you can adjust your growth strategy to match a privacy-friendly approach. So yes, you can run a successful growth team despite the above challenges. 
+
+So, let’s look at what this looks like for Appwrite.
+
+## Privacy-friendly alternatives for your growth stack
+
+Our growth philosophy and rules enable us to grow without relying on tracking you across the web. To give you some insight, here are three (of the six) growth statements we follow for success:
+
+- To center gaining access, knowledge, and information about Appwrite as the most crucial goal to lead to growth.
+- To let information flow freely for developers to find. (accessibility is key)
+- To use paid advertising and sponsorships only to amplify the reach of knowledge and information. (aka creating awareness)
+
+The above can simply be conveyed as having a good content and social media strategy, and the success of this can be analyzed in a privacy-friendly way. We can measure success in metrics like traffic, impressions, social reach, new accounts, etc. Here are some ways we keep the team informed:
+
+- **UTM data** helps us track attributions from different campaigns, mediums, sources, and specific content. We can tell whether a visitor is coming from a specific link or not, for which we currently use [dub.co](https://dub.co/), an open-source alternative to Bitly. This helps us determine whether sponsoring a newsletter was effective or if a piece of content brought us traffic. But that is about as far as we will go. As soon as you leave our website, we lose all your information and would like to keep it that way.
+- **Open source analytics alternatives.** We use privacy-friendly alternatives like [Plausible](https://plausible.io/about) to help us analyze our website and Console data. It is an open-source analytics platform with one of the highest [privacy standards](https://plausible.io/privacy) I have seen. It helps us get demographic and user behavior data in a privacy-friendly manner.
+- **First-party data.** We get a lot from users, like emails and names from authentication. We use it for login purposes, so you don’t have to enter your login details over and over again and add them to our CRM, HubSpot, for email communications. But with this data also comes great responsibility, and with our recent GDPR compliance announcement, we are not only upholding strict data privacy but also certified with the European General Data Protection Regulation (GDPR) standards.
```

Comment:
Doesnt sound like proper beginning of sentence

Reply:
`but also certified with the European General ...`
`but are also certified with the ...`

I think adding `are` is grammatically correct. Please check with someone from US/UK, or Grammarly.